src/lib/containers/QueryVisualizationWrapper.tsx

```diff
@@ -160,6 +176,20 @@ export function QueryVisualizationWrapper(
     [columnAliases],
   )
 
+  const NoContentPlaceholder = useCallback(() => {
+    switch (noContentPlaceholderType) {
+      case NoContentPlaceholderType.INTERACTIVE:
+        if (hasResettableFilters) {
+          return <SearchResultsNotFound />
+        } else {
+          return <ThisTableIsEmpty />
```

Comment:
This is possibly weird-if someone loads an interactive card container with an empty table, it will say "This table is empty". I think it's ok, though. Cards were never shown, so a user probably has no idea that the UI wasn't going to be a table.